mithril.js

```diff
@@ -1959,9 +1988,17 @@
 		}
 	}
 
+	function isNativeError(e) {
```

Comment:
Another drive-by. It was originally in the failed v0.2.1, and since disappeared.